requirements.txt

```diff
@@ -29,4 +29,5 @@ pyairtable==2.3.3
 types-requests==2.32.0.20240907
 s3fs==2024.10.0
 pyarrow==18.1.0
-pyathena==3.9.0
\ No newline at end of file
+pyathena==3.9.0
+asana<5.0.0
```

Comment:
would this mean a major 6 could be installed exposing us to breaking changes? I'd rather lock it to a specific one version

Reply:
fixed!

I've pinned the version to the highest matching package version.